vendor/github.com/avinetworks/sdk/go/models/event_log.go

```diff
@@ -56,7 +56,7 @@ type EventLog struct {
 
 	// Number of report_timestamp.
 	// Required: true
-	ReportTimestamp *int64 `json:"report_timestamp"`
+	ReportTimestamp *string `json:"report_timestamp"`
```

Comment:
Is this something that was added as part of `go mod vendor` or manually edited?

Reply:
This is a manual change. The issue has been reported to concerned team.